src/components/Card/Card.tsx

```diff
@@ -68,11 +65,8 @@ class Card extends React.Component<Props, State> {
           {icon}
         </ResilientImage>
         <div className={gradient}>
-          <div className={minorText} style={{ fontSize: minorSize }}>
-            {minor}
-          </div>
-          <div className={titleText} style={{ fontSize: titleSize }}>
-            {title}
+          <div className={titleText}>
+            <span>{icon} {title}</span>
```

Comment:
Please get rid of the space between `{icon}` and `{title}`

Reply:
I'm happy to remove, but just want to point out that I was simply replicating the usage from voa-zh, here:
https://github.com/Voiceofamerica/voa-zh/pull/3/files

Reply:
ah, true; I guess we can see what it looks like with the space, and go from there